app/views/startpage/startbutton-v1.html

```diff
@@ -0,0 +1,125 @@
+        <div class="govuk-!-padding-bottom-1">&mdash; <a href="#linear">Bulk registrations and linear networks</a><br /></div>
+        <div class="govuk-!-padding-bottom-1">&mdash; <a href="#contact">Contact the Environment Agency</a><br /></div>
+
+<h2 class="govuk-heading-l" id="what">What is a waste exemption?</h2>
+<p>If your business uses, treats, recovers, stores or disposes of waste, you may need an environmental permit.</p>
+<p>However if your waste operation is below specific limits, it may be exempt from needing a permit and can operate under a waste exemption instead.</p>      
+
+<h2 class="govuk-heading-l" id="rules">Waste exemption rules and limits</h2>
+<p>Each exemption has strict limits and conditions. You must check the waste exemption guidance to make sure you can follow them.</p>
+<p>If your waste operation does not meet the limits and conditions of an exemption you will need an environmental permit.</p>
+<p>If you do not follow the rules, your waste operation is not exempt and the Environment Agency will ‘de-register’ your exemption. If you do not meet the limits and conditions of an exemption, or your waste operation has harmed people or the environment, you may be prosecuted.</p>
+     
+<h2 class="govuk-heading-l" id="costs">Exemption costs and renewal</h2>
+<p>Waste exemptions are free except for the T11 WEEE exemption which costs £1,221.</p>
+<p>An exemption registration lasts 3 years. You’ll need to renew the exemption after 3 years if you are still using it.</p>
+
+<h2 class="govuk-heading-l" id="choose">Choose the right exemptions</h2>
+
+<a href="https://www.gov.uk/government/collections/waste-exemptions-storing-waste">Storing waste</a>
+<p>These cover the storage of certain waste materials in secure containers and at secure sites, and storing sludge.</p>
+<a href="https://www.gov.uk/government/collections/waste-exemptions-using-waste">Using waste</a>
+<p>These cover operations such as using waste in construction, spreading certain materials on the land, and burning of waste as fuel.</p>
+<a href="https://www.gov.uk/government/collections/waste-exemptions-using-waste">Disposing of waste</a>
+These cover operations such as depositing sanitary waste, depositing agricultural waste and spoil from dredging inland waters, and burning specific waste in an incinerator or in the open.
+<a href="https://www.gov.uk/government/collections/waste-exemptions-using-waste">Treating waste</a>
+These cover operations such as preparing, recovering, sorting and treating certain waste materials, such as waste wood and plant matter.
+<p class="govuk-body govuk-!-margin-bottom-2">&nbsp;</p>
+
+<h3 class="govuk-heading-m" id="farming">Common farming exemptions</h3>
+<p>These give examples only and are not official advice. You are responsible for selecting the exemptions that are suitable for your waste operation.</p>
+<a href="#">Disposing of waste</a>
```

Comment:
```suggestion
<p><a href="#">Disposing of waste</a></p>
```

This will make the font size normal and add some more spacing between the link and the subsequent header.

Reply:
Also, this link doesn't go anywhere - is that intentional?

Reply:
Yes, we don't have a link for that at the mo :)